Patches/Geth/Ammo_Geth_CE.xml

```diff
@@ -0,0 +1,414 @@
+    </ammoTypes>
+  </CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="SpacerSmallAmmoBase">
+		<defName>Ammo_Geth</defName>
+		<label>Geth thermal clip</label>
+		<description>Thermal clips designed to fit all geth weapons.</description>
+		<statBases>
+			<Mass>0.01</Mass>
+			<Bulk>0.01</Bulk>
+			<MarketValue>0.24</MarketValue>
+		</statBases>
+		<tradeTags>
+                                                <li>CE_AutoEnableTrade_Sellable</li>
+			<li>CE_AutoEnableCrafting_FabricationBench</li>
+			<!-- Plasma ammo can't be handloaded, and the containment cell must be precision-machined anyway -->
+		</tradeTags>
+		<thingCategories>
+			<li>AmmoPlasmaCellRifle</li>
+		</thingCategories>
+		<graphicData>
+			<texPath>ThirdParty/MassEffect/Heatsink</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<ammoClass>Plasma</ammoClass>
+	</ThingDef>
+
+  <!-- ==================== Ammo ========================== -->
+
+  <ThingDef Class="CombatExtended.AmmoDef" Name="RPG7GrenadeBase" ParentName="AmmoBase" Abstract="True">
+    <description>Rocket-propelled missile designed for the ML-77. After being fired a rocket motor kicks in to propel the grenade further.</description>
+    <statBases>
+      <MaxHitPoints>150</MaxHitPoints>
+    </statBases>
+    <tradeTags>
+                <li>CE_AutoEnableTrade_Sellable</li>
+	<li>CE_AutoEnableCrafting_FabricationBench</li>
+    </tradeTags>
+    <thingCategories>
+      <li>AmmoML77CE</li>
+    </thingCategories>
+    <stackLimit>25</stackLimit>
+    <cookOffFlashScale>40</cookOffFlashScale>
+    <cookOffSound>MortarBomb_Explode</cookOffSound>
+  </ThingDef>
+
+  <ThingDef Class="CombatExtended.AmmoDef" ParentName="RPG7GrenadeBase">
+    <defName>Ammo_ML77CE</defName>
+    <label>ML-77 Missile</label>
+    <graphicData>
+      <texPath>Things/Ammo/RPG/HEAT</texPath>
+      <graphicClass>Graphic_StackCount</graphicClass>
+    </graphicData>
+    <statBases>
+      <MarketValue>57.78</MarketValue>
+      <Mass>2.25</Mass>
+      <Bulk>3.5</Bulk>
+    </statBases>
+    <ammoClass>RocketHEAT</ammoClass>
+	<detonateProjectile>Bullet_ML77CE</detonateProjectile>
+  </ThingDef>
+
+	<ThingCategoryDef>
+		<defName>AmmoGeth</defName>
+		<label>Geth thermal clip</label>
+		<parent>AmmoAdvanced</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberPlasmaMedium</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ================== Projectiles ================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseRPG7Grenade">
+		<defName>Bullet_ML77CE</defName>
+		<label>ML77 Missile</label>
+		<graphicData>
+			<texPath>Things/Projectile/Rocket/M72LAW</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+			<shaderType>TransparentPostLight</shaderType>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<speed>24</speed>
```

Comment:
This is probably too slow to be accurate in combat (it's roughly on par with some arrows). I'd at least double it.

Reply:
1 thermal clip is equivalent to a full magazine, 1 ammo isn’t a full clip but a abstract thermal capacity.

Reply:
Done